### ISSUE TITLE
sim_specs['profile'] is used in the manager, but it should be libE_specs['profile'] 

### ISSUE THREAD

**Issue by jmlarson1** [Libensemble/libensemble#548]
The lines here are uncovered:
https://coveralls.io/builds/34607656/source?filename=libensemble%2FlibE_manager.py#L74

**Comment by jlnav** [comment]
As noted on Slack, ``'profile'`` on a glance doesn't appear to be profiling any ``sim_f`` routines, only the following selection from the manager:

```
...
if sim_specs.get('profile'):
    pr = cProfile.Profile()
    pr.enable()  # start profiling

if 'in' not in gen_specs:
        gen_specs['in'] = []

    # Send dtypes to workers
    dtypes = {EVAL_SIM_TAG: hist.H[sim_specs['in']].dtype,
              EVAL_GEN_TAG: hist.H[gen_specs['in']].dtype}
    for wcomm in wcomms:
        wcomm.send(0, dtypes)

    # Set up and run manager
    mgr = Manager(hist, libE_specs, alloc_specs,
                  sim_specs, gen_specs, exit_criteria, wcomms)
    result = mgr.run(persis_info)

    if sim_specs.get('profile'):
        pr.disable()  # stop profiling
...
```

So this is a manager profile? If this feature is kept, should it be moved to ``libE_specs``?

**Comment by jmlarson1** [comment]
Addressed by #549
